core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -932,9 +938,43 @@ public static String getDefaultOncoprintView() {
         }
         return defaultOncoprintView.trim();
     }
-
+    
+    public static boolean useCustomDriverAnnotations() {
+        String useCustomDriverAnnotations = properties.getProperty(USE_CUSTOM_DRIVER_ANNOTATIONS);
```

Comment:
if not set, the default should be true. I think this will do it:
```
String useCustomDriverAnnotations = properties.getProperty(USE_CUSTOM_DRIVER_ANNOTATIONS, "true");
```